boards/lilygo/ttgo_t8c3/doc/index.rst

```diff
@@ -1,12 +1,12 @@
 .. _ttgo_t8c3:
 
-Lilygo TTGO T8-C3
-#################
+Lilygo TTGO T8C3
+################
```

Comment:
Why this change? The commercial name of the board includes the hyphen afaict

Reply:
I was unsure if it should reflect the board name or the target config name. Naming for these products is not consistent: T8C3, T8-C3 and T8_C3 are all used.